src/components/Explore/BalanceSummary.tsx

```diff
@@ -63,32 +70,34 @@ export default function BalanceSummary({ address }: { address: string }) {
             </TotalBalance>
           </TotalBalanceSection>
           <NetworkBalancesSection>Your balances by network</NetworkBalancesSection>
-          <NetworkBalance
-            logoUrl={logoUrl}
-            balance={'1'}
-            tokenSymbol={tokenSymbol ?? 'XXX'}
-            fiatValue={fiatValue}
-            label={label}
-            networkColor={theme.primary1}
-          />
-          <NetworkBalance
-            logoUrl={logoUrl2}
-            balance={'3.3'}
-            tokenSymbol={tokenSymbol ?? 'XXX'}
-            fiatValue={3200}
-            label={'Polygon'}
-            networkColor={TEXT_COLORS['137']}
-          />
+          {data &&
+            Object.entries(data).map(([chainId, amount]) => {
```

Comment:
issue with this option is the keys won't be ordered, so it could change per load. might be better to iterate over a list of chainIds and look into the data object to see if there's anything there